packages/cfd/README.md

```diff
@@ -1,97 +1,45 @@
-    -   [Preview on your local machine](#preview-on-your-local-machine)
--   [Miscellaneous](#Miscellaneous)
--   [Selenium tests](#selenium-tests)
+You need to follow the instructions [here](../../README.md).
 
-## Other documents
+## How To Work With This Project
 
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
+To run and work on this workspace you need to use `npm run serve cfd` command along with `npm run serve core`.
+It will watch `cfd` so after you made a change in this package, it will automatically rebuild `cfd` and recompile `core`.
 
-## Installation
+## Folder Structure
 
-In order to work on your own version of the Deriv Javascript and CSS, please **fork this project**.
-
-You will also need to install the following on your development machine:
-
--   Node.js (10.14.2 or higher is recommended) and NPM (see <https://nodejs.org/en/download/package-manager/>)
--   Go to project root, then run `npm install`
-
-### Use a custom domain
-
-In order to use your custom domain, please put it in a file named `CNAME` inside `scripts` folder of your local clone of deriv-app.
-
-## How to work with this project
-
-### Deploy to your gh-pages for the first time
-
-1.  Register your application [here](https://developers.binary.com/applications/). This will give you the ability to redirect back to your github pages after login.
-    Use `https://YOUR_GITHUB_USERNAME.github.io/deriv-app/` for the Redirect URL and `https://YOUR_GITHUB_USERNAME.github.io/deriv-app/en/redirect` for the Verification URL.
-
-        If you're using a custom domain, replace the github URLs above with your domain and remove the `deriv-app` base path.
-
-2.  In `src/config.js`: Insert the `Application ID` of your registered application in `user_app_id`.
-
-    -   **NOTE:** In order to avoid accidentally committing personal changes to this file, use `git update-index --assume-unchanged src/config.js`
-
-3.  Set `NODE_ENV` to `development` with `export NODE_ENV=development`
-
-4.  Run `npm run deploy:clean`
-
-### Deploy to the root of the gh-pages
-
-This will overwrite modified files and only clear the content of `js` folder before pushing changes. It will leave other folders as they are.
-
-```sh
-npm run deploy
-```
-
-### Clean root and deploy to it
-
-This removes all files and folders and deploys your `dist` folder to the root.
-
-```sh
-npm run deploy:clean
-```
-
-### Deploy to test folder
-
-This will add all your changes to the test folder specified.
-Please ensure it is prefixed with `br_`.
-
-```sh
-npm run deploy:folder "br_my_test_folder"
 ```
+build
+    ├── ...
+    ├── webpack-config.js
+src
+    ├── _common
+    │   ├── ...
+    |   ├── utility.js
+    │
+    ├── Components
+    ├── Constants
+    ├── Containers
+    ├── Helpers
+    ├── Modules
+    |   |──Page404
+    ├── ...
+    ├── Store
+    ├── ...
+    ├── Utils
+    │   ├── Validator
+    |   |   ├──...
+    |   |   ├── validator.js
```

Comment:
Maybe add some comments on what are the purposes of some of the folders - ie how does CFD group functionality?